main.cpp

```diff
@@ -73,12 +25,33 @@ void testPairToNumber(
     assert(pairNumber == expectedPairNumber);
 }
 
+void printColorCodeManualRef(const std::string &filename) {
+    std::ofstream manual(filename);
+    manual << "Color Code Reference Manual\n";
+    manual << "--------------------------------\n";
+
+    for(int major = 0; major < TelCoColorCoder::numberOfMajorColors; ++major) {
+        for(int minor = 0; minor < TelCoColorCoder::numberOfMinorColors; ++minor) {
+            TelCoColorCoder::ColorPair colorPair(static_cast<TelCoColorCoder::MajorColor>(major),
+                                                 static_cast<TelCoColorCoder::MinorColor>(minor));
+            int pairNumber = TelCoColorCoder::GetPairNumberFromColor(static_cast<TelCoColorCoder::MajorColor>(major),
+                                                                     static_cast<TelCoColorCoder::MinorColor>(minor));
+            manual << "Pair Number: " << std::setw(2) << pairNumber << " -> " << colorPair.ToString() << "\n";
+        }
+    }
+
+    manual.close();
+    std::cout << "Reference manual generated: " << filename << std::endl;
+}
+
 int main() {
     testNumberToPair(4, TelCoColorCoder::WHITE, TelCoColorCoder::BROWN);
     testNumberToPair(5, TelCoColorCoder::WHITE, TelCoColorCoder::SLATE);
 
     testPairToNumber(TelCoColorCoder::BLACK, TelCoColorCoder::ORANGE, 12);
     testPairToNumber(TelCoColorCoder::VIOLET, TelCoColorCoder::SLATE, 25);
 
+     printColorCodeManualRef("ColorCodeReferenceManual.txt");
```

Comment:
The above functions have asserts. But this print function's results are not asserted. That means the entire effort of testing its output becomes manual. Can you think of a way to reduce the manual effort?